ui/src/app/core/collection/collection.helper.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020 the original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+export class CollectionHelper {
+	static getCollectionItemUrl(apiKey: string, imageId: string): string {
+		return `api/v1/static/${apiKey}/images/${imageId}`;

```

Comment:
api/v1/ might be a part of environment configuration

Reply:
used environment.userApiUrl